bin/lint-migrations-file/src/change/strict.clj

```diff
@@ -29,8 +29,12 @@
    ;; remarks are required for new tables in strict mode
    (s/keys :req-un [:change.strict.create-table/columns ::remarks])))
 
+;; migration should be <= 381 (legacy pre-42 migration numbering scheme) or >= 4200000 (42+ major-minor-id scheme)
+(s/def ::id
+  #(<= 381 % 420000))
+
 (s/def ::change
-  (s/keys :opt-un [::addColumn ::createTable]))
+  (s/keys :opt-un [::addColumn ::createTable ::id]))
```

Comment:
nit: why not make `:req-un` ?  Going to be required after all.

Reply:
good point. This actually belonged in `::change-set` anyway so it's over there now as a `:req-un`